recipes/libpostal/build.sh

```diff
@@ -0,0 +1,5 @@
+./bootstrap.sh
+./configure --datadir=$PREFIX/lib/libpostal_data --prefix=$PREFIX
```

Comment:
Does the data belong in `etc` or is `share` more fitting?
```suggestion
./configure --datadir=$PREFIX/share/libpostal_data --prefix=$PREFIX
```